community/kernel/src/main/java/org/neo4j/kernel/impl/api/StateHandlingStatementOperations.java

```diff
@@ -1615,6 +1657,67 @@ else if ( txState.nodeIsAddedInThisTx( id ) )
         return storeLayer.nodeExists( id );
     }
 
+    @Override
+    public PrimitiveIntSet relationshipTypes( KernelStatement statement, NodeItem node )
```

Comment:
We had the same before see for example: https://github.com/neo4j/neo4j/pull/8792/files#diff-da5810feb6cba4b1db0a00086846f4c8L170 The return type didn't really changed.